integration/command.go

```diff
@@ -252,6 +254,7 @@ func (c *command) start(t *testing.T) {
 
 	err := c.command.Start()
 	if err != nil {
+		fmt.Printf("Command returned:\n%s\n%s\n", c.stderr.String(), c.stdout.String())
```

Comment:
Do `c.stderr` and `c.stdout` contain data at this point?

Reply:
Yeah, you are right. This change is useless.